src/Metal/MTLComputePassSampleBufferAttachmentDescriptorArray.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Metal {
+
+	public partial class MTLComputePassSampleBufferAttachmentDescriptorArray {
+
+		public MTLComputePassSampleBufferAttachmentDescriptor this[nuint i] {
```

Comment:
where `[Get|Set]Object` introduced in the same OS version as the type ?

Reply:
Yes, new class, new methods.